chapters/en/slides/module1/module1_01.md

```diff
@@ -101,39 +195,45 @@ and review code.
         </ul>
     </div>
     <div class="column">
-        <img src="https://code.visualstudio.com/assets/images/code-stable.png" alt="VS Code Logo" width="200px"></img>
+        <img src="/module1/vscode-icon.png" alt="VS Code Logo" width="200px"></img>
     </div>
 </div>
 
 Notes:
-You are already familiar with JupyterLab
-from the Programming with Python course,
-where we used it to edit our Jupyter notebooks.
+As a soft prerequisite for this course,
```

Comment:
Yea, I was thinking that it would be beneficial that they still have a little bit of Jupyter knowledge beforehand, but I reviewed Module 7 now and I agree with you that we cover it in enough detail there also for a beginner. It would be good if they could have a little bit of it in module 1 already as we discussed before, I have opened #71 for this